frontend/sass/norent/_letter-builder.scss

```diff
@@ -88,6 +88,10 @@ html[data-safe-mode-no-js] .jf-norent-internal-above-footer-content {
     }
   }
 
+  .jf-letter-translation ul li {
+    font-weight: inherit;
+  }
+
```

Comment:
Ew, I don't like it when I have to "undo" another rule's styling just to get something to look like one would expect it to... At least it's right next to the rule it's overriding though!

Reply:
Ugh yeah that is annoying to have to do. I've tried using the `:not()` selectors as much as possible for things like that but that seems very hard to try and maneuver in this scenario!